constantine/ethereum_verkle_ipa.nim

```diff
@@ -215,7 +217,7 @@ func deserialize*(dst: var EthVerkleIpaMultiProof,
 # TODO: refactor, this shouldn't use curves_primitives but internal functions
 import ./lowlevel_fields
 
-func mapToBaseField*(dst: var Fp[Banderwagon],p: EC_TwEdw[Fp[Banderwagon]]) =
+func mapToBaseField*(dst: var Fp[Banderwagon],p: EC_TwEdw[Fp[Banderwagon]]) {.libPrefix: prefix_ipa, discardable.} =
```

Comment:
Those should be renamed `map_to_base_field` so the C API uses proper snake-case. For existing Nim code it will have no regression due to being flexible on the casing.